### PR TITLE
fix(broken): Manhuasy CryptoJS-protected; Daprob is Apache mod_autoindex (remove)

### PR DESCRIPTION
**Pull request by AgentKush**
## Summary

Two adjacent changes that emerged from the same probing session, packaged together because they share the same diagnosis pattern (sites that look like Madara on the surface but aren't actually scrapeable Madara underneath).

1. **Update `Manhuasy` (www.manhuasy.com) `@Broken` reason** — site is alive and serves a real Madara skin, but the chapter images are protected by the `wp-manga-chapter-images-protection` WordPress plugin (CryptoJS-encrypted images, same as `CatharsisWorld` already labels). The current annotation "layout/API changed, needs rewrite" is misleading — the layout is fine, the chapter image delivery is encrypted. Changing the reason to match `CatharsisWorld`'s honest description tells future maintainers exactly what they're walking into.

2. **Remove `Daprob` (daprob.com)** — looks like a Madara WP frontend at the homepage (10 cards with `div.page-item-detail`, Madara CSS classes), but every detail page redirects into a literal **Apache `mod_autoindex` directory listing** of static `capitulo-N.html` files. Not actually a Madara site under the hood, no admin-ajax, no pagination, no search — just a frozen 10-item catalog with static-HTML chapter delivery. The maintainer's earlier static-HTML-snapshot intuition was right; remove the dead Madara skeleton.

### Manhuasy — Madara front, CryptoJS chapter image protection

```
GET /                                            200, 13 page-item-detail cards visible
GET /manhua/                                     200, 11 cards on /manhua/
GET /manhua/?m_orderby=latest                    200, 11 cards (different slugs — sort works)
GET /?s=&post_type=wp-manhua                     200, 13 cards
GET /manhua/mr-qiao-madam-spoiled-by-you/        200, 174063 bytes — substantive detail page
```

The detail page contains the WP-Manga plugin assets:
- `wp-content/plugins/wp-manga-chapter-images-protection/assets/css/dots.css`
- `wp-content/plugins/wp-manga-chapter-images-protection/assets/css/plugin.css`

This is the same CryptoJS chapter-image-protection plugin already documented on `CatharsisWorld` (`@Broken("Site migrated to WP-Madara with wp-manga-chapter-images-protection plugin (CryptoJS-encrypted chapter images)")`). The current Manhuasy annotation "layout/API changed, needs rewrite" is misleading — anyone trying a min-diff fix will discover that browse works, search works, sort works, but the chapter-image bytes are encrypted blobs that need CryptoJS-aware page rendering. Match the `CatharsisWorld` precedent for accuracy.

### Daprob — Madara skin over Apache static index

```
GET /                                  200, 10 page-item-detail cards (Madara skin homepage)
GET /?s=&post_type=wp-manga            200, same 10 cards
GET /page/2/                           HTTP 404
GET /page/2/?s=&post_type=wp-manga     HTTP 404
GET /page/3/                           HTTP 404
GET /manga/                            200, 8629B
POST /wp-admin/admin-ajax.php          HTTP 404
```

But — `GET /manga/martial-inverse/` (a sample slug from the homepage) returns a **1783-byte literal Apache directory listing**:

```html
<title>Index of /manga/martial-inverse</title>
<h1>Index of /manga/martial-inverse</h1>
<a href="/manga/">Parent Directory</a>
<a href="capitulo-247.html">capitulo-247.html</a>  2026-03-11 15:40  644
<a href="capitulo-248.html">capitulo-248.html</a>  2026-03-11 15:40  644
<a href="capitulo-262.html">capitulo-262.html</a>  2026-03-11 15:40  644
...
```

That's `mod_autoindex` output, not a WP-Madara detail page. The "site" is a frozen-snapshot Apache static-file tree behind a Madara-styled homepage. Browse via `/page/N/` doesn't exist; admin-ajax doesn't exist; search doesn't exist; chapters are literal `.html` files in a directory tree. Catalog is 10 items, all detail pages are autoindex.

The current `MadaraParser` cannot work against this — the URL contract (`/wp-admin/admin-ajax.php` for browse, `/manga/<slug>/` Madara detail page, inline `<li class="wp-manga-chapter">`) doesn't match anything the site serves. A custom rewrite would be a tiny 10-item-only parser of marginal value. Cleaner to remove and let someone re-add it as a fresh parser if a new operator picks up the brand.

## What changes

`Manhuasy.kt`:
```diff
-@Broken("Site is online but parser is broken — layout/API changed, needs rewrite")
+@Broken("Site uses wp-manga-chapter-images-protection plugin (CryptoJS-encrypted chapter images)")
```

`Daprob.kt`: removed.

`docs/parsers.json`: regenerated (1162 / 346).

## 5 QCs

**Vulnerability:** none. Annotation rewrite + leaf-class delete; no public API changed.

**Quality:** evidence-driven. Manhuasy probe surfaces the exact same `wp-manga-chapter-images-protection/assets/css/{dots,plugin}.css` references that `CatharsisWorld` references in its annotation — these CSS files come bundled with the WordPress plugin that delivers AES/CryptoJS-encrypted chapter image bytes to the browser-side decryptor. Daprob probe surfaces literal `<title>Index of /manga/martial-inverse</title>` Apache `mod_autoindex` HTML, not Madara — a hard category mismatch.

**Bug risk:** very low. Manhuasy stays `@Broken`, just with a more accurate reason. Daprob is a leaf class with no other source-tree references (`grep -rln Daprob src/main/kotlin` confirms only the file being removed).

**Concern:** one — if a maintainer later writes CryptoJS chapter-image support, both `Manhuasy` and `CatharsisWorld` (same plugin) would benefit; the updated annotation makes that pairing discoverable.

**Compatibility:** zero impact. Other parsers untouched; KSP regenerates `MangaParserSource` enum (drops `DAPROB`).

## Test plan

- [x] Manhuasy probe: `/`, `/manhua/`, `/manhua/?m_orderby=latest`, `/?s=&post_type=wp-manhua` all 200; detail page references `wp-manga-chapter-images-protection` plugin assets.
- [x] Daprob probe: `/page/N/` HTTP 404 for all N; admin-ajax 404; `/manga/<slug>/` returns Apache `mod_autoindex` listing.
- [x] `git grep Daprob src/main/kotlin` — only the file being removed references the class.
- [x] `python3 docs/build_catalog.py` — 1162 / 346 (-1 parser, -1 broken).
- [ ] CI runs `./gradlew kspKotlin` — KSP regenerates enum without `DAPROB`.

### CODE REVIEW

**Comment by AgentKush** [comment]
Closing for redo. Daprob removal needs re-verification against non-Madara frameworks (it might be a static FoolSlide-style snapshot reachable by a different base). Manhuasy reason update can be re-filed once the audit confirms CryptoJS is the only blocker.
